### DISCUSSION TITLE
Minor Update to Documentation - onfiguration.adoc

### DISCUSSION THREAD

**Discussion by jpartain89** [keycloak/keycloak#33292] [Ideas]
The explanation over the config priorties is currently listing the final variable wrong. I'm simply submitting an update to fix that incorrectly listed variable:

From: 

> If `--db-url=cliValue` had not been used, the applied value would be `KC_DB_URL=envVarValue`. If the value were not applied by either the command line or an environment variable, `db-url=confFileValue` would be used. If none of the previous values were applied, the value `kc.db-url=confFileValue` would be used due to the lowest priority among the available configuration sources.

to:

> If `--db-url=cliValue` had not been used, the applied value would be `KC_DB_URL=envVarValue`. If the value were not applied by either the command line or an environment variable, `db-url=confFileValue` would be used. If none of the previous values were applied, the value `kc.db-url=keystoreValue` would be used due to the lowest priority among the available configuration sources.

With the specified update changing `kc.db-url=confFileValue` to `kc.db-url=keystoreValue` at the end.

**Comment by codespearhead** [comment]
I've just created a PR [1] to address this.

[1] https://github.com/keycloak/keycloak/pull/33489

Could you [mark this comment as the answer](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer) so it doesn't show up as unanswered in the GitHub discussions tab?
